scripts/sct_maths.py

```diff
@@ -172,14 +177,15 @@ def get_parser():
     filtering = parser.add_argument_group("FILTERING METHODS")
     filtering.add_argument(
         "-smooth",
-        metavar='',
+        metavar=Metavar.list,
+        type=list_type(',', float),
         help='Gaussian smoothing filter with specified standard deviations in mm for each axis (Example: 2,2,1) or '
              'single value for all axis (Example: 2).',
         required=False)
     filtering.add_argument(
         '-laplacian',
-        nargs="+",
-        metavar='',
+        metavar=Metavar.list,
+        type=list_type(',', float),
         help='Laplacian filtering with specified standard deviations in mm for all axes (Example: 2).',
```

Comment:
This example should show the syntax you need to use the list type. And while we're here, does it mean you need to give as many values as axes? So 3 or 4 depending on the image? Except there's a special case if you only give one value?

It's worded difficultly.

```suggestion
        help='Laplacian filtering with specified standard deviations in mm for all axes (Example: 2,3,2).',
```

??

Reply:
I agree this is confusing. I think the error message later in the file is much clearer about what this means:

https://github.com/neuropoly/spinalcordtoolbox/blob/d443363b1028fe5fc653abd8c3b06b8af9913d9d/scripts/sct_maths.py#L311

I'll update the help description.